client/src/pages/private/ParticipantTable.js

```diff
@@ -379,14 +379,22 @@ const ParticipantTable = () => {
     );
 
     if (isMoH || isHA || isSuperUser) {
-      // should be able to see all users in both MHAW/ HCA programs
-      setPrograms(allPrograms);
+      if (
+        isHA &&
+        !['region_vancouver_island', 'region_interior'].some((region) => roles.includes(region))
```

Comment:
Can you pop this array in a const at the top?

Reply:
Done for BE and FE